### ISSUE TITLE
EntityTooSmall during test_multipart_mix

### ISSUE THREAD

**Issue by gaul** [s3fs-fuse/s3fs-fuse#1304]
When debugging some Alibaba OSS changes, I found that s3fs creates a too small 16-byte part during `test_multipart_mix` which fails due to `EntityTooSmall`:

```
s3fs: [INF]       curl.cpp:ParallelMixMultipartUploadRequest(1678): Upload Part [tpath=/testrun-7148/big-file-s3fs.txt-mix][start=0][size=10485760][part=1]
s3fs: [INF]       curl.cpp:UploadMultipartPostSetup(3881): [tpath=/testrun-7148/big-file-s3fs.txt-mix][start=0][size=10485760][part=1]
s3fs: [INF]       curl.cpp:ParallelMixMultipartUploadRequest(1678): Upload Part [tpath=/testrun-7148/big-file-s3fs.txt-mix][start=10485760][size=10485760][part=2]
s3fs: [INF]       curl.cpp:UploadMultipartPostSetup(3881): [tpath=/testrun-7148/big-file-s3fs.txt-mix][start=10485760][size=10485760][part=2]
s3fs: [INF]       curl.cpp:ParallelMixMultipartUploadRequest(1678): Upload Part [tpath=/testrun-7148/big-file-s3fs.txt-mix][start=20971520][size=16][part=3]
s3fs: [INF]       curl.cpp:UploadMultipartPostSetup(3881): [tpath=/testrun-7148/big-file-s3fs.txt-mix][start=20971520][size=16][part=3]
s3fs: [INF]       curl.cpp:ParallelMixMultipartUploadRequest(1700): Copy Part [tpath=/testrun-7148/big-file-s3fs.txt-mix][start=20971536][size=5242864][part=4]
...
s3fs: <Error>
s3fs:   <Code>EntityTooSmall</Code>
s3fs:   <Message>Your proposed upload smaller than the minimum allowed size.</Message>
s3fs:   <RequestId>5ED8C7A228D3F934395F00B6</RequestId>
s3fs:   <HostId>oss-ap-northeast-1.aliyuncs.com</HostId>
s3fs:   <ETag>DA7F19812CF4C9F6B129B70B4A7F901E</ETag>
s3fs:   <MinSizeAllowed>102400</MinSizeAllowed>
s3fs:   <PartNumber>3</PartNumber>
s3fs:   <ProposedSize>16</ProposedSize>
s3fs:   <UploadId>54E7E793A22745A99C0135FBBBC601B1</UploadId>
s3fs: </Error>
```

**Comment by gaul** [comment]
I don't think this is related, but Alibaba's part limits differ:

> Each part has a part number that identifies the part. Part numbers can range from 1 to 10000. The size of each part ranges from 100 KB to 5 GB in size. In multipart upload mode, each part except the last part must be greater than 100 KB in size.

From: https://www.alibabacloud.com/help/doc-detail/31993.htm#reference-pnq-2px-wdb

**Comment by ggtakec** [comment]
This issue is probably resolved in #1313.
